plugins/woocommerce/client/blocks/assets/js/atomic/blocks/product-elements/sale-badge/block.json

```
@@ -10,11 +10,20 @@
 		"isDescendentOfSingleProductTemplate": {
 			"type": "boolean",
 			"default": false
+		},
+		"isPreview": {
+			"type": "boolean",
+			"default": false
 		}
 	},
 	"usesContext": [ "query", "queryId", "postId" ],
 	"keywords": [ "WooCommerce" ],
 	"textdomain": "woocommerce",
 	"apiVersion": 3,
+	"example": {
+		"attributes": {
+			"isPreview": false
+		}
+	},
```

```suggestion
	"example": {},
```
As suggested above, the above should suffice unless I am missing something.

---

I am not sure why didn't work initially when testing it without the attribute 🤷 . Tested it again and works as you mentioned. Thanks for pointing it out!